src/renderers/WebGLRenderer.js

```diff
@@ -2080,47 +2080,90 @@ function WebGLRenderer( parameters = {} ) {
 
 	};
 
-	this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level = 0 ) {
+	this.copyTextureToTexture = function ( srcTexture, dstTexture, dstPosition = undefined, srcBox = undefined, dstLevel = 0, srcLevel = 0 ) {
+
+		if ( ! srcTexture.isTexture ) {
+
+			console.warn( 'THREE.WebGLRenderer: .copyTextureToTexture() now expects (srcTexture, dstTexture, dstPosition, srcBox, dstLevel, srcLevel).' );
+			return this.copyTextureToTexture( arguments[ 1 ], arguments[ 2 ], arguments[ 0 ], undefined, arguments[ 3 ] );
+
+		}
+
+		const image = srcTexture.mipmaps[ srcLevel ] || ( srcLevel == 0 ? srcTexture.image : null );
+		if ( ! image ) {
+
+			console.error( 'mipmap level ', srcLevel, ' is not available in srcTexture' );
+			return;
+
+		}
+
+		let skipPixels = srcBox ? srcBox.min.x : 0;
+		let skipRows = srcBox ? srcBox.min.y : 0;
+		let x = dstPosition ? dstPosition.x : 0;
+		let y = dstPosition ? dstPosition.y : 0;
+
+		// ensure skipPixels, skipRows, x and y >= 0
+		if ( skipPixels < 0 || x < 0 ) {
+
+			const delta = Math.min( skipPixels, x );
+			x -= delta;
+			skipPixels -= delta;
+
+		}
+
+		if ( skipRows < 0 || y < 0 ) {
+
+			const delta = Math.min( skipRows, y );
+			y -= delta;
+			skipRows -= delta;
+
+		}
+
+		const width = Math.min( dstTexture.image.width - x, srcBox ? Math.min( image.width, srcBox.max.x + 1 ) - skipPixels : image.width );
```

Comment:
If we decide to remove the above two conditions, then we also need to update that part because it will not be valid anymore